.gitignore

```diff
@@ -297,3 +297,8 @@ test/PackagedCommands/Consumers/*/project.json
 
 # VS generated files
 launchSettings.json
+
+# Generated project.json files (the project.json.template file is the authoritative copy)
+setuptools/independent/DepsProcessor/project.json
+build_projects/dotnet-host-build/project.json
+build_projects/update-dependencies/project.json
```

Comment:
Would it be best to remove the project.jsons we have checked in now to avoid confusion?